components/supervisor/frontend/src/index.ts

```diff
@@ -77,6 +77,14 @@ const toStop = new DisposableCollection();
         return;
     }
 
+    // @Devspaces: Add some data expected by vscode 1.78. We don't currently support all endpoints at the moment.
+    // TODO: Remove when the platform is fully synced with newer gitpod release.
+    window.gitpod.loggedUserID = gitpodServiceClient.user.id;
+    window.gitpod.openDesktopIDE = () => console.error('Open Desktop IDE is not supported by the current gitpod backend'); // Unsupported
```

Comment:
@daniel-wachira I assume we could connect via IDE but not via browser

Reply:
Yes. This option is new and adds a "open desktop IDE" option in the browser menu. The existing method should still work just fine.